index.html

```diff
@@ -100,18 +112,20 @@ <h3 class="sub-heading-style">Avatar Fallback</h3>
                         <div class="avatar-fb">SM</div>
                         <div class="avatar-fb">RS</div>
                         <div class="avatar-fb">HS</div>
-                        <img src="./images/avatars/fallback-avatars/man.png" alt="man">
-                        <img src="./images/avatars/fallback-avatars/woman.png" alt="woman">
-                        <img src="./images/avatars/fallback-avatars/user.png" alt="user">
+                        <img class="circular-avatar" src="./images/avatars/fallback-avatars/man.png" alt="man">
+                        <img class="circular-avatar" src="./images/avatars/fallback-avatars/woman.png" alt="woman">
+                        <img class="circular-avatar" src="./images/avatars/fallback-avatars/user.png" alt="user">
                     </div>
                     <div class="code-block-style">
                         <iframe
                             src="https://carbon.now.sh/embed/LrYz9ggrzfc5MunBWLks"
-                            style="width: 748px; height: 250px; border:0; transform: scale(1); overflow:hidden;"
+                            style="border:0; transform: scale(1); overflow:hidden;"
```

Comment:
```suggestion
                            style="border:0; transform: scale(1); overflow:hidden;"
```
The common styling from iframe component can be moved to a different class as it's getting reused